tools/flaky_test_monitor/common/testdata/test_data.go

```diff
@@ -19,197 +20,160 @@ type Level1TestData struct {
-			Elapsed:    0,
-			Output: []struct {
-				Item string "json:\"item\""
-			}{
-				{Item: "=== RUN   " + name + "\n"},
-				{Item: "--- PASS: " + name + " (0.00s)\n"},
-			},
-		},
+func getPassedTestPackage(name string, packageName string) common.Level1TestResult {
+	return common.Level1TestResult{
+		CommitSha:  COMMIT_SHA,
+		RunID:      RUN_ID,
+		CommitDate: getCommitDate(),
+		JobRunDate: getJobRunDate(),
+		Test:       name,
+		Package:    packageName,
+		Pass:       1,
+		Exception:  0,
+		Elapsed:    0,
 	}
-	return row
 }
 
-func getPassedTestElapsed(name string, elapsed float32, elapsedStr string) common.Level1TestResultRow {
-	row := common.Level1TestResultRow{
-		TestResult: common.Level1TestResult{
-			CommitSha:  COMMIT_SHA,
-			CommitDate: getCommitDate(),
-			JobRunDate: getJobRunDate(),
-			Test:       name,
-			Package:    CRYPTO_HASH_PACKAGE,
-			Result:     "1",
-			NoResult:   false,
-			Elapsed:    elapsed,
-			Output: []struct {
-				Item string "json:\"item\""
-			}{
-				{Item: "=== RUN   " + name + "\n"},
-				{Item: "    --- PASS: " + name + " (" + elapsedStr + "s)\n"},
-			},
-		},
+func getPassedTestElapsed(name string, elapsed float64, elapsedStr string) common.Level1TestResult {
+	return common.Level1TestResult{
+		CommitSha:  COMMIT_SHA,
+		RunID:      RUN_ID,
+		CommitDate: getCommitDate(),
+		JobRunDate: getJobRunDate(),
+		Test:       name,
+		Package:    CRYPTO_HASH_PACKAGE,
+		Pass:       1,
+		Exception:  0,
+		Elapsed:    elapsed,
 	}
-	return row
 }
 
-func getPassedTestPackageElapsedOutput(name string, packageName string, elapsed float32, elapsedStr string, output string) common.Level1TestResultRow {
+func getPassedTestPackageElapsedOutput(name string, packageName string, elapsed float64, elapsedStr string, output string) common.Level1TestResult {
 	row := getPassedTestElapsedOutput(name, elapsed, elapsedStr, output)
-	row.TestResult.Package = packageName
+	row.Package = packageName
 	return row
 }
 
-func getPassedTestElapsedOutput(name string, elapsed float32, elapsedStr string, output string) common.Level1TestResultRow {
-	row := common.Level1TestResultRow{
-		TestResult: common.Level1TestResult{
-			CommitSha:  COMMIT_SHA,
-			CommitDate: getCommitDate(),
-			JobRunDate: getJobRunDate(),
-			Test:       name,
-			Package:    CRYPTO_HASH_PACKAGE,
-			Result:     "1",
-			NoResult:   false,
-			Elapsed:    elapsed,
-			Output: []struct {
-				Item string "json:\"item\""
-			}{
-				{Item: "=== RUN   " + name + "\n"},
-				{Item: output},
-				{Item: "--- PASS: " + name + " (" + elapsedStr + "s)\n"},
-			},
-		},
+func getPassedTestElapsedOutput(name string, elapsed float64, elapsedStr string, output string) common.Level1TestResult {
+	return common.Level1TestResult{
+		CommitSha:  COMMIT_SHA,
+		RunID:      RUN_ID,
+		CommitDate: getCommitDate(),
+		JobRunDate: getJobRunDate(),
+		Test:       name,
+		Package:    CRYPTO_HASH_PACKAGE,
+		Pass:       1,
+		Exception:  0,
+		Elapsed:    elapsed,
```

Comment:
why is the expected output removed? it's passed as a parameter and should be part of the generated level 1 test result